tests/conditional_compilation/test_cc.py

```diff
@@ -78,38 +81,47 @@ def test_minimized_pkg(test_id, model, openvino_root_dir, artifacts):  # pylint:
 
 
 @pytest.mark.dependency(depends=["cc_collect", "minimized_pkg"])
-def test_infer(test_id, model, artifacts):
+def test_infer(test_id, models, artifacts):
     """Test inference with conditional compiled binaries."""
     out = artifacts / test_id
     minimized_pkg = out / "install_pkg"
-    return_code, output = run_infer(model, f"{out}_cc.npz", minimized_pkg)
+    infer_out_dir_cc = artifacts / f"{test_id}_cc" / "inference_result"
+
+    return_code, output = run_infer(models, infer_out_dir_cc, minimized_pkg)
     assert return_code == 0, f"Command exited with non-zero status {return_code}:\n {output}"
 
 
 @pytest.mark.dependency(depends=["cc_collect", "minimized_pkg"])
-def test_verify(test_id, model, openvino_ref, artifacts, tolerance=1e-6):  # pylint: disable=too-many-arguments
+def test_verify(test_id, models, openvino_ref, artifacts, tolerance=1e-6):  # pylint: disable=too-many-arguments
     """Test verifying that inference results are equal."""
     out = artifacts / test_id
     minimized_pkg = out / "install_pkg"
-    out_file = f"{out}.npz"
-    out_file_cc = f"{out}_cc.npz"
-    return_code, output = run_infer(model, out_file, openvino_ref)
+
+    infer_out_dir_cc = artifacts / f"{test_id}_cc" / "inference_result"
```

Comment:
May be easier to use following implementation?
`infer_out_dir_cc = out / "inference_result_cc/"`
`infer_out_dir = out / "inference_result/"`